assembly/src/parsers/io_ops/local_ops.rs

```diff
@@ -8,12 +8,12 @@ use vm_core::utils::PushMany;
 
 /// Pushes the first element of the word at the specified local procedure memory index onto the
 /// stack.
-pub fn parse_push_local(
+pub fn parse_load_local(
```

Comment:
We can actually do this operation much more efficiently now using `mload` operation (instead of using `mloadw` + some stack manipulation).